src/pages/explore/Explore.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Navbar from "../../components/navbar/Navbar";
+import Sidebar from "../../components/sidebar/Sidebar";
```

Comment:
Can use barrel exports here involving making of an index.js file within components that exports Navbar and Sidebar from there making it easy for you to import multiple components in a single line!!